lib/chusaku/routes.rb

```diff
@@ -108,13 +113,13 @@ def backfill_routes(routes)
       # Given a route, extract the controller and action strings.
       #
       # @param {ActionDispatch::Journey::Route} route - Route instance
-      # @return {Array<String>} - [String, String]
-      def extract_controller_and_action_from(route)
-        defaults = route.defaults
-        controller = defaults[:controller]
-        action = defaults[:action]
+      # @return {Array<Object>} - [String, String, Hash]
```

Comment:
BTW, do you folow some particular agreements on documentation tag syntax? If it is YARD's (at least that's what `@tags` imply), they can be a bit different:
1. Avoid `-` between declaration and explanation, YARD does think about it as a "list start": https://rdoc.info/gems/chusaku/Chusaku/Routes#call-class_method
2. For a constant list (tuple), YARD [has](https://yardoc.org/types.html) agreement that you can just write `Array(String, String, Hash)`, or even `(String, String, Hash)`

Reply:
Ah, this is very helpful, I appreciate these references. 🙇🏻  I wasn't following any standards with these comments, but now I will in a follow up PR.